tree/ntuple/v7/inc/ROOT/RNTupleSerialize.hxx

```diff
@@ -69,7 +68,7 @@ public:
    static constexpr DescriptorId_t kZeroFieldId = std::uint64_t(-2);
 
    struct REnvelopeLink {
-      std::uint32_t fUnzippedSize = 0;
+      std::uint64_t fUnzippedLength = 0;
```

Comment:
Instances of `fUnzippedLenght` also go against the convention set in the specification. Since `Length` already encodes the information that the data is uncompressed in memory, then `Unzipped` becomes redundant.